swssconfig/swssconfig.cpp

```diff
@@ -39,12 +52,36 @@ void dump_db_item(KeyOpFieldsValuesTuple &db_item)
     SWSS_LOG_DEBUG("]");
 }
 
-bool write_db_data(vector<KeyOpFieldsValuesTuple> &db_items)
+bool is_zmq_table(string table_name, int zmq_flag)
```

Comment:
>table_name [](http://example.com/codeflow?start=25&length=10)

Suggest using a conffiles to list all the ZMQ applicable table names. It is more future-proof and also easy to implement. No need to define ZMQ enable feature fla

Reply:
Will improve this by read from CONFIG_DB

Reply:
Fixed, will test this change with https://github.com/sonic-net/sonic-buildimage/pull/22414

Reply:
Testing generates config file by feature flag:
https://github.com/sonic-net/sonic-buildimage/pull/22451

If it works, will update this PR.